### ISSUE TITLE
Validation loss curve never goes down when training stackex tasks

### ISSUE THREAD

**Issue by gcarba** [snap-stanford/relbench#137]
I have not been able to properly train the GNN with the provided dataset (stackex) with tasks 'engage' and 'votes'. The training loss curve decreases as expected, but the validation curve stays horizontal or goes slightly upwards if I increase the number of epochs. Is this behavior something that you see on your end as well?
Note: my last download was on April 4th from master.

**Comment by rishabh-ranjan** [comment]
We have updated the repository significantly since then. Hopefully, it is easier to get things running now and you don't face the same problems.
